### ISSUE TITLE
[BUG] Can't build glasstron with node-gyp

### ISSUE THREAD

**Issue by AzuKaOwO** [Lightcord/Lightcord#129]
npm test failed saying delet C:\Users\--------\Lightcord\node_modules\electron
and install manualy 
and i do everything but when installing manualy

C:\Users\-----\Lightcord\node_modules\glasstron>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! find VS
gyp ERR! find VS msvs_version not set from command line or npm config
gyp ERR! find VS VCINSTALLDIR not set, not running in VS Command Prompt
gyp ERR! find VS could not use PowerShell to find Visual Studio 2017 or newer
gyp ERR! find VS looking for Visual Studio 2015
gyp ERR! find VS - not found
gyp ERR! find VS not looking for VS2013 as it is only supported up to Node.js 8
gyp ERR! find VS
gyp ERR! find VS **************************************************************
gyp ERR! find VS You need to install the latest version of Visual Studio
gyp ERR! find VS including the "Desktop development with C++" workload.
gyp ERR! find VS For more information consult the documentation at:
gyp ERR! find VS https://github.com/nodejs/node-gyp#on-windows
gyp ERR! find VS **************************************************************
gyp ERR! find VS
gyp ERR! configure error
gyp ERR! stack Error: Could not find any Visual Studio installation to use
gyp ERR! stack     at VisualStudioFinder.fail (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:121:47)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:74:16
gyp ERR! stack     at VisualStudioFinder.findVisualStudio2013 (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:351:14)
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:70:14
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\find-visualstudio.js:372:16
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\util.js:54:7
gyp ERR! stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\util.js:33:16
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:315:5)
gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
gyp ERR! stack     at maybeClose (internal/child_process.js:1048:16)
gyp ERR! System Windows_NT 10.0.19042
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\-------\Lightcord\node_modules\glasstron
gyp ERR! node -v v14.15.1
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! glasstron@0.0.4 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the glasstron@0.0.4 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\----------\AppData\Roaming\npm-cache\_logs\2020-12-07T20_22_45_720Z-debug.log

C:\Users\---------\Lightcord>

note:i aready installed visual studio code

[2020-12-07T20_22_45_720Z-debug.log](https://github.com/Lightcord/Lightcord/files/5655136/2020-12-07T20_22_45_720Z-debug.log)

**Comment by JeanOUINA** [comment]
I recommend doing what they say on https://github.com/nodejs/node-gyp#on-windows.
TL;DR: run `npm install --global windows-build-tools` in an administrator powershell/cmd, then retry.

(Also if you just want to use Lightcord, you're probably looking for a [windows download](https://github.com/Lightcord/lightcord/releases/latest))

**Comment by AzuKaOwO** [comment]
> I recommend doing what they say on https://github.com/nodejs/node-gyp#on-windows.
> TL;DR: run `npm install --global windows-build-tools` in an administrator powershell/cmd, then retry.
> 
> (Also if you just want to use Lightcord, you're probably looking for a [windows download](https://github.com/Lightcord/lightcord/releases/latest))

k ty im trying
